killercoda/http-scaler/install-kedify-agent.md

```diff
@@ -1,15 +1,39 @@
-# Kedify Agent
+The Kedify agent is an application designed to simplify deployment and management of KEDA seamlessly within your Kubernetes environment. For the purposes of the scenario here in Killercoda, you will use the pre-installed [Kedify kubectl plugin](https://github.com/jkremser/kubectl-kedify). You can click on the command below, it will get automatically executed in the terminal on the right part of the screen. Enter `y`{{}} to indicate "yes" to the installation prompt.
+```bash
+kubectl kedify install --email http-scaler@killercoda.com
```

Comment:
@jkremser is there an option to automatically aprove (eg `-y` option)?

Reply:
@wozniakjan I would like to somehow get emails of people that try this, can we try similar style that we are doing for Quickstart? Ie users will need to submit their emails? WDYT?

Reply:
I've [added](https://github.com/jkremser/kubectl-kedify/commit/3d713f4daae74e595d6d9d5ac157cfad0d9bd0c3#diff-ded5ef3cf4698b5d6d514b6c23af336b128a94af22452963a3bec7d1dd0cfe83R103) a check there that finds out if it's running in a interactive shell or not and if not, it shouldn't ask the final question. Not sure if it works in killercoda though. It can be figured out by:

```
echo $-
```

and if the `i` is there or not. If it doesn't work there, I can add the `-y` option

Reply:
the shell in killercoda is actually also interactive, the copy & paste there is done in the UI frontend. But we probably want to leave out the prepopulated email so some interaction with the user will be there nevertheless